Makefile.include

```diff
@@ -530,7 +531,8 @@ endif
 
 # Generate machine readable RIOT VERSION macro
 RIOT_VERSION_CODE ?= $(shell echo ${RIOT_VERSION} | \
-                      sed -E 's/([0-9]+).([0-9]+).?([0-9]+)?.*/RIOT_VERSION_NUM\\\(\1,\2,0\3,${RIOT_EXTRAVERSION}\\\)/')
+                      sed -E 's/([0-9]+).([0-9]+).?([0-9]+)?.*/RIOT_VERSION_NUM\\\(\1,\2,0\3,${RIOT_EXTRAVERSION}\\\)/' | \
```

Comment:
```suggestion
                      sed -E 's/([0-9]+)\.([0-9]+)\.?([0-9]+)?.*/RIOT_VERSION_NUM\\\(\1,\2,0\3,${RIOT_EXTRAVERSION}\\\)/' | \
```

Otherwise things like `6035-HEAD` also match (as it is currently the case in the `release-tests`, since `.` resolves to "any character" ;-)